Example/Tryouts/TRYExampleAppDelegate.m

```diff
@@ -51,8 +51,8 @@ - (void)motionRecognizingWindowDidRecognizeShakeMotion:(TRYMotionRecognizingWind
         return;
     }
 
-    [Tryouts presentFeedBackControllerFromViewController:topMostController
-                                                animated:YES];
+    [Tryouts presentFeedBackControllerWithPresentingViewController:topMostController
```

Comment:
Let's please change this method's name to `presentFeedbackControllerFromViewController:`

Note that the "B" in Feedback is not uppercase. If the class itself has it in uppercase, also change that.